priv/static/Linux.sh

```diff
@@ -15,145 +15,145 @@ HOME="${HOME:-$(eval echo ~$USER)}"
-            ;;
-        "erlang")
-            sudo apt-get update
-            sudo apt-get -y install build-essential autoconf m4 libncurses5-dev libwxgtk3.0-gtk3-dev libwxgtk-webview3.0-gtk3-dev libgl1-mesa-dev libglu1-mesa-dev libpng-dev libssh-dev unixodbc-dev xsltproc fop libxml2-utils libncurses-dev openjdk-11-jdk
-            asdf plugin add erlang https://github.com/asdf-vm/asdf-erlang.git
-            asdf install erlang 25.1.2 && asdf global erlang 25.1.2
-            ;;
-        "elixir")
-            asdf plugin add elixir https://github.com/asdf-vm/asdf-elixir.git
-            asdf install elixir 1.14.2-otp-25
-            asdf global elixir 1.14.2-otp-25
-            asdf reshim elixir 1.14.2-otp-25
-            ;;
-        "phoenix")
-            source ~/.bashrc >/dev/null 2>&1
-            source ~/.zshrc >/dev/null 2>&1
-            mix local.hex --force
-            echo "y" | mix archive.install hex phx_new 1.7.0-rc.2
-            ;;
-        "nodejs")
-            asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
-            asdf install nodejs 16.17.0
-            asdf global nodejs 16.17.0
-            asdf reshim nodejs 16.17.0
-            ;;
-        "postgresql")
-            sudo apt-get update
-            sudo apt-get -y install linux-headers-generic build-essential libssl-dev libreadline-dev zlib1g-dev libcurl4-openssl-dev uuid-dev icu-devtools
-            asdf plugin add postgres https://github.com/smashedtoatoms/asdf-postgres.git
-            asdf install postgres 15.1
-            asdf global postgres 15.1
-            ;;
-        "vim")
-            sudo apt-get install -y vim
-            ;;
-        "chrome")
-            sudo wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-            sudo apt install -y ./google-chrome-stable_current_amd64.deb
-            ;;
-        "chromedriver")
-            source ~/.bashrc >/dev/null 2>&1
-            source ~/.zshrc >/dev/null 2>&1
-            npm install -g chromedriver
-            ;;
-        "docker")
-            sudo apt-get update
-            sudo apt-get install -y \
+    "Zsh")
+        sudo apt-get install -y zsh
+        ;;
+    "oh-my-zsh")
+        sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)" "" --unattended
+        ;;
+    "wget")
+        sudo apt-get install -y wget
+        ;;
+    "Homebrew")
+        NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+        echo '# Set PATH, MANPATH, etc., for Homebrew.' >>/home/"$(whoami)"/.zprofile
+        echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"' >>/home/"$(whoami)"/.zprofile
+        eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+
+        # recommendation after install homebrew
+        brew install gcc
+        ;;
+    "Asdf")
```

Comment:
lowercase

Reply:
noted, btw I use order that reflect the installation messages